### DISCUSSION TITLE
Tracking and redirecting 404s

### DISCUSSION THREAD

**Discussion by tmsndrs** [wagtail/wagtail#11385] [Ideas]
It would be great if Wagtail presented admin with a list of urls that responded with 404 errors (and how many times they were accessed) with a quick way of adding redirects so that if I see /abuotus is coming up a lot I can add a redirect to /aboutus etc.

**Comment by kaedroho** [comment]
Should we also add the ability to add a redirect when deleting/moving/changing slug of a page? Also, if you delete a page with redirects pointing to it, those redirects would be deleted as well (and the editor would not be aware of this).

https://github.com/torchbox/wagtail/issues/925
